conf/code-lists/holder-of-authorisation-codes/holder-of-authorisation-codes.json

```diff
@@ -19,220 +34,190 @@
     "en": "Authorised Economic Operator - Security and safety",
     "cy": "Gweithredwr Economaidd Awdurdodedig - Diogelwch"
   },
-  {
-    "code": "MIB",
-    "en": "Merchandise in Baggage",
-    "cy": "Nwyddau mewn Bagiau"
-  },
   {
     "code": "APEX",
     "en": "Approved Exporter",
-    "cy": "Allforiwr sydd wedi\u00e2\u0080\u0099i gymeradwyo"
+    "cy": "Allforiwr sydd wedi\u2019i gymeradwyo"
+  },
+  {
+    "code": "AWB",
+    "en": "Authorised Weigher of Bananas",
+    "cy": "Y person sydd wedi\u2019i awdurdodi i bwyso bananas"
+  },
+  {
+    "code": "BOI",
+    "en": "Binding Origin Information",
+    "cy": "Gwybodaeth am Darddiad sy\u2019n Rhwymo"
+  },
+  {
+    "code": "BTI",
+    "en": "Binding Tariff Information",
+    "cy": "Gwybodaeth am Dariff sy\u2019n Rhwymo"
+  },
+  {
+    "code": "CCL",
+    "en": "Authorisation for centralised clearance",
+    "cy": "Awdurdodiad ar gyfer clirio canoledig"
   },
   {
     "code": "CGU",
     "en": "Customs comprehensive guarantee",
     "cy": "Gwarant gynhwysfawr y tollau"
   },
+  {
+    "code": "CSDR",
+    "en": "Community System of Duty Reliefs",
+    "cy": "System Gymunedol o Ryddhad Tollau"
+  },
   {
     "code": "CSE",
     "en": "Customs Supervised Exports",
     "cy": "Allforion Dan Oruchwyliaeth Tollau"
   },
   {
-    "code": "FP",
-    "en": "Freeports Special Procedure",
-    "cy": "Gweithdrefn Arbennig Porthladdoedd Rhydd"
+    "code": "CVA",
+    "en": "Simplification of the determination of amounts being part of the customs value of goods",
+    "cy": "Symleiddio\u2019r penderfyniad ar y symiau sy\u2019n rhan o werth tollau nwyddau"
   },
   {
     "code": "CW1",
     "en": "Operation of storage facilities for the customs warehousing of goods in a public customs warehouse type I",
-    "cy": "Gweithrediad cyfleusterau storio ar gyfer tollau\u00e2\u0080\u0099n cadw nwyddau mewn warws tollau cyhoeddus math I"
+    "cy": "Gweithrediad cyfleusterau storio ar gyfer tollau\u2019n cadw nwyddau mewn warws tollau cyhoeddus math I"
```

Comment:
Asking just to be on the safe side... is this conversion equivalent?

Reply:
I decided to leave this matter for another time. I rolled back the original version.